engines/scumm/script_v6.cpp

```diff
@@ -1424,7 +1424,7 @@ void ScummEngine_v6::o6_getAnimateVariable() {
 	 		_currentRoom == ((_game.id == GID_BASEBALL2001) ? 4 : 3) && \
 			vm.slot[_currentScript].number == 2105 && \
 			a->_costume == ((_game.id == GID_BASEBALL2001) ? 107 : 99) && \
-			readVar(0x8000 + 5) != 0)
+			readVar(0x8000 + 5) != 0 && readVar(0x8000 + 22) == 4)
```

Comment:
Could you please add a comment here to explain what this is doing?

Reply:
Added in 8c51752